packages/api/src/routes/medical/dtos/patientDTO.ts

```diff
@@ -5,7 +5,7 @@ import { DemographicsDTO } from "./demographicsDTO";
 export type PatientDTO = {
   facilityIds: string[];
   externalId?: string;
-  dateCreated?: Date;
+  createdAt?: Date;
```

Comment:
I know I asked about why not using `createdAt`, but this is changing the DTO which changes the API contract and SDK.

Maybe we can use `createdAt` to the newly created properties instead of exposing it to the API for now? In the API v2 we can revisit this.

Reply:
reverted this change